packages/grafana-ui/src/saga-themes/tokens/core.json

```diff
@@ -0,0 +1,893 @@
+{
+  "dimension": {
+    "scale": {
+      "value": "8px",
+      "type": "dimension",
+      "description": "Base grid unit to be used as multiplier for sizing, spacing and other scales."
+    }
+  },
+  "spacing": {
+    "0": {
+      "value": "{dimension.scale}*0",
+      "type": "spacing"
+    },
+    "25": {
+      "value": "{dimension.scale}*0.25",
+      "type": "spacing"
+    },
+    "50": {
+      "value": "{dimension.scale}*0.5",
+      "type": "spacing"
+    },
+    "75": {
+      "value": "{dimension.scale}*0.75",
+      "type": "spacing"
+    },
+    "100": {
+      "value": "{dimension.scale}*1",
+      "type": "spacing"
+    },
+    "150": {
+      "value": "{dimension.scale}*1.5",
+      "type": "spacing"
+    },
+    "200": {
+      "value": "{dimension.scale}*2",
+      "type": "spacing"
+    },
+    "250": {
+      "value": "{dimension.scale}*2.5",
+      "type": "spacing"
+    },
+    "300": {
+      "value": "{dimension.scale}*3",
+      "type": "spacing"
+    },
+    "400": {
+      "value": "{dimension.scale}*4",
+      "type": "spacing"
+    },
+    "500": {
+      "value": "{dimension.scale}*5",
+      "type": "spacing"
+    },
+    "600": {
+      "value": "{dimension.scale}*6",
+      "type": "spacing"
+    },
+    "800": {
+      "value": "{dimension.scale}*8",
+      "type": "spacing"
+    },
+    "1000": {
+      "value": "{dimension.scale}*10",
+      "type": "spacing"
+    }
+  },
+  "breakpoint": {
+    "xs": {
+      "value": "0px",
+      "type": "sizing",
+      "description": "XS breakpoint min-width value."
+    },
+    "sm": {
+      "value": "544px",
+      "type": "sizing",
+      "description": "SM breakpoint min-width value."
+    },
+    "md": {
+      "value": "769px",
+      "type": "sizing",
+      "description": "MD breakpoint min-width value. (1 more than regular ipad in portrait)"
+    },
+    "lg": {
+      "value": "992px",
+      "type": "sizing",
+      "description": "LG breakpoint min-width value."
+    },
+    "xl": {
+      "value": "1200px",
+      "type": "sizing",
+      "description": "XL breakpoint min-width value."
+    },
+    "xxl": {
+      "value": "1440px",
+      "type": "sizing",
+      "description": "XXL breakpoint min-width value."
+    }
+  },
+  "font": {
+    "fontFamily": {
+      "primary": {
+        "value": "Inter",
+        "type": "fontFamilies"
+      },
+      "monospace": {
+        "value": "Roboto Mono",
+        "type": "fontFamilies"
+      }
+    },
+    "fontWeight": {
+      "thin": {
```

Comment:
We don't have all of these fonts imported in code for Inter, just regular and medium, if we want to support all these we need to import more of those. But I think we should still double check if we want all these to be used in grafana. Same with all the fontSize and lineHeights. I guess the 'core' ones don't matter since no one should be using these directly? But we will be doing a representation of all the tokens in Storybook or Saga and then it will seem like we support 20 different font sizes

Reply:
I find it useful to have them mapped in our core tokens as they're the base of the Inter font family.
We're not using all of them of course at the moment, but for the sake of scalability, theming and customisation in the future, that's something we can benefit from already having in our new structure. Regarding these core tokens, it is not intended for designers or engineers to use them freely or use them at all, they are for internal use as references to our semantic-level tokens. Once again, about well-structured scales that provide theming and customisation possibilities for the future, they must be created properly from the beginning so that in the future you avoid having to add steps into those scales that will impact the locations where the core tokens are being used as aliases already.
No matter if we have all of those possibilities in the core tier, we will show in our reference site and storybook what people can and should be using.